src/components/structures/MatrixChat.js

```diff
@@ -361,6 +365,7 @@ module.exports = React.createClass({
                         room_id: result.room_id,
                         event_id: payload.event_id,
                         invite_sign_url: payload.invite_sign_url,
+                        room_oob_data: payload.room_oob_data,
```

Comment:
ok, but things read the `view_room` payload and look for `oob_data`, not `room_oob_data`. This one looks like a bug rather than just being inconsistent.